### PR TITLE
fix: clean up components directory + fix Storybook

### PR DESCRIPTION
**Pull request by therealemjy**
## Jira ticket(s)

VEN-2932

## Changes

- Upgrade Storybook to v8.x
- Add stories to components that didn't have any
- Remove `withCenterStory` in favor of a applying global styling to stories
- Move `ApproveDelegateSteps` component into Market page directory
- Fix `ApproveDelegateSteps` component’s property types
- Move `AssetWarning` component into Market page directory
- Fix text displayed on `Countdown` component when showing days
- Add `inline-flex` class to `Pill` component
- Move `Form`, `AddTokentToWalletButton`, `ProposalCard`, `Redirect` components into containers directory

### CODE REVIEW

File: apps/evm/src/pages/Market/Market/AssetWarning/testIds.ts
Comment (gleiser-oliveira) [inline apps/evm/src/pages/Market/Market/AssetWarning/testIds.ts:1]: I think there is an extra 'Market' folder here

File: apps/evm/src/pages/Proposal/__tests__/index.spec.tsx [hunk 1]
```diff
@@ -96,7 +97,8 @@ describe('Proposal page', () => {
         data: { message: 'Fake error message' },
       }),
     }));
-    const { getByText } = renderComponent(<Proposal />);
+    const { getByText, debug } = renderComponent(<Proposal />);
+    debug();
```
Comment (gleiser-oliveira) [inline apps/evm/src/pages/Proposal/__tests__/index.spec.tsx]: Looks like it's unintentional

Comment (therealemjy) [reply apps/evm/src/pages/Proposal/__tests__/index.spec.tsx]: Correct, I removed it. Thanks

File: apps/evm/src/pages/Market/Market/AssetWarning/testIds.ts
Comment (therealemjy) [reply apps/evm/src/pages/Market/Market/AssetWarning/testIds.ts:1]: The naming is confusing: there's a Market component inside the Market page directory... I've renamed the Market child component to Page to make things clearer.

**Review by gleiser-oliveira — APPROVED** [review]

**Pull request merged**